web/beacon-app/src/features/auth/components/Register/SuccessfullAccountCreation.tsx

```diff
@@ -26,6 +29,22 @@ function SuccessfullAccountCreation() {
     };
   }, [userEmail, navigateTo, storage]);
 
+  const resendEmailHandler = useCallback(() => {
+    resendEmail(userEmail as string);
+  }, [userEmail, resendEmail]);
+
+  useEffect(() => {
+    if (resendResult) {
+      toast.dismiss();
```

Comment:
i think this can be omitted as we don't have any other modal open.

Reply:
The toast message appeared twice, if I clicked the resend button more than once. https://www.awesomescreenshot.com/video/21098762?key=443ffc227799d9899e65deaddb2dc873

After adding `toast.dismiss()` I didn't see that happen again.

Reply:
I've added `reset` instead.